src/parsers/perf/perfparser.cpp

```diff
@@ -999,6 +999,27 @@ class PerfParserPrivate : public QObject
 
     void addSample(const Sample& sample)
     {
+        auto& lastTime = m_lastSampleTimePerCore[sample.cpu];
+        if (lastTime) {
+            if (frequencyResult.cores.size() <= sample.cpu) {
+                frequencyResult.cores.resize(sample.cpu + 1);
+            }
+            frequencyResult.cores[sample.cpu].coreName = QString::number(sample.cpu);
```

Comment:
this is bogus, why not simply use the cpu number directly? what is this used for?

Reply:
it isn't used